cranelift-codegen/meta/src/isa/x86/recipes.rs

```diff
@@ -2795,6 +2853,25 @@ pub fn define<'shared>(
                     "#,
                 ),
         );
+
+        recipes.add_template_recipe(
+            EncodingRecipeBuilder::new("icscc_id_ref", f_unary, 5 + 3)
```

Comment:
Since icscc_i{b,d}_ref are unguarded by predicates, at least one of the associated encodings has to be unreachable in the encodings interpreter. Can you check this (= only one of the two recipes is actually used), and if that's true, remove this one? (which takes more memory)

Reply:
Talked to @sunfishcode and we decided to create a new recipe `is_zero` that uses less memory. Both icscc_i{b,d}_ref are gone now. :)